content/outcomes/cloud-native-development/general__using_cloud_foundry.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Using Cloud Foundry
```

Comment:
Again, consider having this page be about PAS in general, with this content as a sporting article of some kind.

Reply:
Updated!